packages/cli/lib/lib/webpack/webpack-client-config.js

```diff
@@ -120,13 +120,9 @@ function getBabelEsmPlugin(config) {
 				beforeStartExecution: (plugins, newConfig) => {
 					const babelPlugins = newConfig.plugins;
 					newConfig.plugins = babelPlugins.filter(plugin => {
-						if (
-							Array.isArray(plugin) &&
-							plugin[0].indexOf('fast-async') !== -1
-						) {
-							return false;
-						}
-						return true;
+						return !(
+							Array.isArray(plugin) && plugin[0].indexOf('fast-async') !== -1
+						);
```

Comment:
This is totally unrelated but I noticed it when fixing this.

Reply:
can we keep it in a separate PR for better tracking?

Reply:
Sure, sorry.

Reply:
It's been reverted.